core/controlplane/config/templates/cloud-config-controller

```diff
@@ -1692,43 +1697,54 @@ write_files:
           apiGroup: rbac.authorization.k8s.io
 {{ end }}
 
-  - path: /etc/kubernetes/manifests/kube-proxy.yaml
+  - path: /srv/kubernetes/manifests/kube-proxy-ds.yaml
     content: |
-        apiVersion: v1
-        kind: Pod
+        apiVersion: extensions/v1beta1
```

Comment:
apps/v1beta1 :)

Reply:
Actually, according to the API reference, the `apiVersion` for the `DaemonSet` object is `extensions/v1beta1` for Kubernetes v1.7. When we upgrade to v1.8, then we'll need to change this to `apps/v1beta2` (if I read the documentation correctly).